spynnaker8/models/projection.py

```diff
@@ -114,7 +115,10 @@ def _check_population_param(self, param, connector):
         if isinstance(param, PopulationView):
             if (isinstance(connector, OneToOneConnector) or
                     isinstance(connector, AllToAllConnector) or
-                    isinstance(connector, FixedProbabilityConnector)):
+                    isinstance(connector, FixedProbabilityConnector) or
+                    isinstance(connector, FixedNumberPostConnector) or
+                    isinstance(connector, FixedNumberPreConnector) or
+                    isinstance(connector, FixedTotalNumberConnector)):
```

Comment:
With this many possibles to check I would suggest and Asbstract Class for the ones that support views.

PS. isinstance mthod second param is "A type or a class, or a tuple of types and/or classes"

Reply:
Abstract class added for connectors that support views on machine.